### DISCUSSION TITLE
Can applications get pen data from this driver?

### DISCUSSION THREAD

**Discussion by yaw-man** [OpenTabletDriver/OpenTabletDriver#2383] [Q&A] [answered]
Hi,

I've written a couple tablet-aware applications using the Wintab API.
I'd like to support more tablets and operating systems.
Does OpenTabletDriver provide a cross-platform way to get pressure-sensitive pen data from a tablet?

**Comment by X9VoiD** [comment, marked as answer]
Yes, there's the [OpenTabletDriver nuget package](https://www.nuget.org/packages/OpenTabletDriver) for that. Currently there's not much documentation on how to use OTD's public API, and there's only two projects that make use of it right now that I'm aware of, [osu! framework](https://github.com/ppy/osu-framework) and OTD itself via OpenTabletDriver.Daemon, so you're on your own with that for now.

**Reply by yaw-man** [reply]
Thank you!
I forgot to specify that my project is built with CMake and written in unmanaged C++, so I probably won't be able to use this, but I really appreciate the information!
